core/scripts/install.sh

```diff
@@ -98,6 +100,12 @@ if ($options['root'] && is_dir($options['root'])) {
 }
 unset($options['root']);
 
+// Set the site URL.
+if ($options['url']) {
+  $_SERVER['HTTP_HOST'] = $options['url'];
```

Comment:
Isn't HTTP_HOST the domain name only, without the protocol? So if `$options['url']` was `https://example.com`, I think `HTTP_HOST` would just be `example.com`. Perhaps this should be checked with `parse_url()` to pull that out.

Also, multisite can be configured not just on domains but also sub-directories. Are those handled in the PR as well?

Reply:
Good idea. I've updated the code to parse URL strings better.

Re. sub-directories, are you referring to `$sites['8080.www.example.com.mysite.test'] = 'example';`? If so, I've never used this and am not sure how to go about allowing for that scenario...